test/integration/associations/belongs-to-many.test.js

```diff
@@ -206,7 +206,11 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), function() {
       return self.sequelize.dropAllSchemas().then(function() {
         return self.sequelize.createSchema('acme');
       }).then(function() {
-        return self.sequelize.sync({force: true});
+        return AcmeUser.sync({force: true});
```

Comment:
The models that are attached to the sequelize object to do not have the schema set anymore since .schema no longer sets the option for all models but now creates a copy of the model with the $schema attribute set. sequelize.define will only addd the base model with the default schema. An alternative would be to set the schema attribute on the define but then that wouldn't be using .schema at all.